src/nvim/terminal.c

```diff
@@ -615,15 +620,16 @@ bool terminal_enter(void)
   curwin->w_p_siso = 0;
 
   adjust_topline(s->term, buf, 0);  // scroll to end
-  // erase the unfocused cursor
-  invalidate_terminal(s->term, s->term->cursor.row, s->term->cursor.row + 1);
   showmode();
   curwin->w_redr_status = true;  // For mode() in statusline. #8323
   redraw_custom_title_later();
-  ui_busy_start();
+  ui_cursor_shape();
```

Comment:
for GUIs should news.txt mention something like "terminal-mode no longer signals 'busy' event" ? cc @fredizzimo

Reply:
AFAICT this was only being used to hide the cursor in the TUI. I'm not sure how GUIs might have been using it though.